src/track/beats.h

```diff
@@ -420,8 +420,8 @@ class Beats : private std::enable_shared_from_this<Beats> {
     mixxx::audio::FrameDiff_t endBeatLengthFrames() const;
 
     std::vector<BeatMarker> m_markers;
-    mixxx::audio::FramePos m_endMarkerPosition;
-    mixxx::Bpm m_endMarkerBpm;
+    mixxx::audio::FramePos m_lastMarkerPosition;
```

Comment:
can you add an explanatory comment here to avoid this discussion again:  "// This is the position of the last marker in the track. For many tracks, this may be the first beat of the track."

Reply:
https://github.com/mixxxdj/mixxx/pull/4510/commits/74eb1bfc365b54e7b14aa3559f6e61c00a6c0366